pkg/pwgen/memorable.go

```diff
@@ -4,16 +4,34 @@ import "strings"
 
 // GenerateMemorablePassword will generate a memorable password
 // with a minimum length
-func GenerateMemorablePassword(minLength int, symbols bool) string {
+func GenerateMemorablePassword(minLength int, symbols bool, capitals bool) string {
 	var sb strings.Builder
+	var upper = false
 	for sb.Len() < minLength {
-		sb.WriteString(randomWord())
+		if capitals {
+			if randomInteger(2) == 0 {
+				sb.WriteString(strings.Title(randomWord()))
+				upper = true
+			} else {
+				sb.WriteString(randomWord())
+			}
+		} else {
+			sb.WriteString(randomWord())
+		}
```

Comment:
the double if with both having the same else clause could be rewritten:
```suggestion
		// when requesting uppercase, we randomly uppercase words
		if capitals && randomInteger(2) == 0 {
			sb.WriteString(strings.Title(randomWord()))
			upper = true
		} else {
			sb.WriteString(randomWord())
		}
```